proto/storage/image_integration.proto

```diff
@@ -124,6 +125,12 @@ message GoogleConfig {
     bool wif_enabled       = 4;
 }
 
+message ScannerV4Config {
+    int32  num_concurrent_scans = 1;
```

Comment:
do we plan on making this configurable? just curious

Reply:
That was the assumption yes, don't believe we have metrics showing its usage for the current scanner however.